go/consensus/tendermint/apps/staking/transactions.go

```diff
@@ -404,6 +405,15 @@ func (app *stakingApplication) reclaimEscrow(ctx *api.Context, state *stakingSta
 		}
 	}
```

Comment:
Perhaps also add logging (like for `AddEscrow`) along these lines:

```go
	ctx.Logger().Debug("ReclaimEscrow: started debonding stake",
		"to", toAddr,
		"from", reclaim.Account,
		"base_units", stakeAmount,
		"active_shares", reclaim.Shares,
		"debonding_shares", debondingShares,
	)
```

Reply:
Done.